sql/src/test/scala/org/apache/sedona/sql/functionTestScala.scala

```diff
@@ -134,14 +134,15 @@ class functionTestScala extends TestBaseScala with Matchers with GeometrySample
       var polygonDf = sparkSession.sql("select ST_GeomFromWKT(polygontable._c0) as countyshape from polygontable")
       polygonDf.createOrReplaceTempView("polygondf")
       val polygon = "POLYGON ((110.54671 55.818002, 110.54671 55.143743, 110.940494 55.143743, 110.940494 55.818002, 110.54671 55.818002))"
-      val forceXYExpect = "POLYGON ((471596.69167460164 6185916.951191288, 471107.5623640998 6110880.974228167, 496207.109151055 6110788.804712435, 496271.31937046186 6185825.60569904, 471596.69167460164 6185916.951191288))"
+      // Floats don't have an exact decimal representation. String format varies across jvm:s. Do an approximate match.
```

Comment:
This fixes builds on java 11. The string representation of floats changed between java 8 and java 11.

There are other tests in sedona-core that fail on java 11 but this fixes sedona-sql